account_analytic_distribution_manual/models/account_analytic_distribution_manual.py

```diff
@@ -0,0 +1,28 @@
+from odoo import _, api, fields, models
+
+
+class AccountAnalyticDistributionManual(models.Model):
+    _name = "account.analytic.distribution.manual"
+    _inherit = "analytic.mixin"
+    _description = "Account analytic distribution manual"
+
+    name = fields.Char(required=True)
+    active = fields.Boolean(default=True)
+    company_id = fields.Many2one(
+        "res.company", required=True, default=lambda self: self.env.company
+    )
+
```

Comment:
When creating Analytic Manual distribution, should we control that percentage cannot be greather than 100?
Please check this example:

<img width="1055" alt="image" src="https://github.com/user-attachments/assets/522e7b7f-881a-41a8-9c30-a5317e0a9411">

Reply:
Odoo does not have this constraint in analytic models. @pedrobaeza , what do you think?

Reply:
It's OK as is. It's on analytic distributed nature to allow more than 100%.